### PR TITLE
Config with pure python

### PR DESCRIPTION
**Pull request by icemac**
* Lint the code.
* Fix deprecation warnings.
* Add support for Python 3.9.

### CODE REVIEW

File: src/zope/app/container/browser/tests/test_contents_functional.py [hunk 1]
```diff
@@ -202,41 +227,49 @@ def test_inplace_change_title(self):
...
 
-
     def test_pasteable_for_deleted_clipboard_item(self):
         """Tests Paste button visibility when copied item is deleted."""
 
         root = self.getRootFolder()
         root['foo'] = File()    # item to be copied/deleted
         root['bar'] = File()    # ensures that there's always an item in
-                                # the collection view
+        # the collection view
```
Comment (sallner) [inline src/zope/app/container/browser/tests/test_contents_functional.py:236]:
```suggestion
# ensures that there's always an item in the collection view
root['bar'] = File()
```
This way it seems to transfer the intention better.

File: src/zope/app/container/browser/tests/test_adding.py [hunk 2]
```diff
@@ -13,46 +13,49 @@
...
 """
+from zope.testing import renormalizing
+import re
 import doctest
 import unittest
 
 import zope.interface
 import zope.security.checker
+from zope.browsermenu.interfaces import AddMenu
+from zope.browsermenu.interfaces import IMenuItemType, IBrowserMenu
```
Comment (sallner) [inline src/zope/app/container/browser/tests/test_adding.py:24]: We do not have `isort` activated here for now, or am I missing some configuration?

**Review by sallner — APPROVED** [review]
Looks good with minor comments.

File: src/zope/app/container/browser/tests/test_adding.py (hunk 2 above)
Comment (icemac) [reply src/zope/app/container/browser/tests/test_adding.py:24]: No we haven't. I think this was done by `autopep8`.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**